### DISCUSSION TITLE
Spun-out Standard mod causes PP Counter to underflow if first object is a spinner

### DISCUSSION THREAD

**Discussion by ItsJustSomeDude** [ppy/osu#16402] [Q&A] [answered]
Add the PP counter to your skin or use default.  Enable the spun-out mod.  Start a map that has a spinner as the first object.  Let the spinner start spinning and watch the PP counter go negative brrrrr.

Maps I know you can try with:
https://osu.ppy.sh/beatmapsets/436611#osu/940378
https://osu.ppy.sh/beatmapsets/646291#osu/1369388

Vid:
https://user-images.githubusercontent.com/46335020/148844661-1574ddaa-8f30-4516-9ef5-670f1402bf47.mp4

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16404
